backend/codesirius/problems/models/language.py

```
@@ -16,5 +16,9 @@ class Language(BaseModel):
     name = models.CharField(max_length=50, unique=True)
     version = models.CharField(max_length=50, blank=True)  # Optional
 
+    class Meta:
+        verbose_name = "Language"
+        verbose_name_plural = "Languages"
+
     def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)